<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketClient</class>
 <widget class="QWidget" name="SocketClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_ip">
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ip">
         <property name="inputMask">
          <string>000.000.000.000</string>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_port">
         <property name="styleSheet">
          <string notr="true">margin-left:3px;</string>
         </property>
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_port">
         <property name="inputMask">
          <string extracomment="1200 - 65535"/>
         </property>
         <property name="text">
          <string>12345</string>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QWidget" name="horizontalWidget1" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_text"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sendText">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_networkStatus">
        <property name="styleSheet">
         <string notr="true">margin-top:3px;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Connection Statusï¼šnone&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_disconnect">
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
